src/components/App/App.js

```diff
@@ -1,14 +1,17 @@
 import styles from "./App.module.css";
 import Header from "../Header/Header";
 import Lead from "../Lead/Lead";
-import PhotoGrid from "../PhotoGrid/PhotoGrid";
+import Place from "../Place/Place";
+// import PhotoGrid from "../PhotoGrid/PhotoGrid";
```

Comment:
We do not have commented code in the code

Reply:
fixed.